pages/Styles-Overview.md

```diff
@@ -13,10 +13,13 @@ The `polygons` draw style tessellates and extrudes vector shapes into 3D geometr
 The `lines` draw style can turn either polygonal or line data into lines.
 
 #### `points`
-The `points` draw style draws a square at a point, and can fill the point with either a customizable dot or a `sprite`. It can work with point data, lines, or polygons. Points will "collide" with each other, with only the winner being drawn, determined by the `[priority](draw.md#priority)` draw parameter.
+The `points` draw style draws a square at a point, and can fill the point with either a customizable dot or a `sprite`. It can work with point data, lines, or polygons. Points will "collide" with each other, with only the winner being drawn, determined by the [`priority`](draw.md#priority) draw parameter.
```

Comment:
Saying it's a "square" might be a bit confusing... I mean technically it is a square that can be filled with anything via a shader, but it's a circle point by default. Minor point though.